composer.json

```diff
@@ -19,23 +19,24 @@
         }
     ],
     "require": {
-        "symfony/dependency-injection": "^5.3|^6.0",
-        "symfony/framework-bundle": "^5.3|^6.0",
-        "symfony/http-client": "^5.3|^6.0",
-        "symfony/routing": "^5.3|^6.0",
-        "symfony/security-bundle": "^5.3|^6.0",
-        "symfony/http-foundation": "^5.3|^6.0",
+        "php": ">=8.1",
+        "symfony/dependency-injection": "^6.4|^7.0",
+        "symfony/framework-bundle": "^6.4|^7.0",
+        "symfony/http-client": "^6.4|^7.0",
+        "symfony/routing": "^6.4|^7.0",
+        "symfony/security-bundle": "^6.4|^7.0",
+        "symfony/http-foundation": "^6.4|^7.0",
         "knpuniversity/oauth2-client-bundle": "^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0",
-        "symfony/expression-language": "^5.3|^6.0",
-        "symfony/finder": "^5.3|^6.0",
-        "symfony/form": "^5.3|^6.0",
-        "symfony/stopwatch": "^5.3|^6.0",
-        "symfony/twig-bundle": "^5.3|^6.0",
-        "symfony/validator": "^5.3|^6.0",
-        "symfony/yaml": "^5.3|^6.0"
+        "phpunit/phpunit": "^10.5",
+        "symfony/expression-language": "^6.4|^7.0",
+        "symfony/finder": "^6.4|^7.0",
+        "symfony/form": "^6.4|^7.0",
+        "symfony/stopwatch": "^6.4|^7.0",
+        "symfony/twig-bundle": "^6.4|^7.0",
+        "symfony/validator": "^6.4|^7.0",
+        "symfony/yaml": "^6.4|^7.0"
```

Comment:
Symfony 5.4 is still officially supported and for a while. I think using a BC Layer is not too expensive here. I have no idea what @konandrum thinks about this...